### ISSUE TITLE
Alerts Annotation on Graph : preserve value when editing, AnnotationList: Include alerts annotation in the list

### ISSUE THREAD

**Issue by ZealousMacwan** [grafana/grafana#20469]
**What would you like to be added**:

Save alert title, values and time in the annotation description.
Auto-add "tags" on alerts annotation. 
Include auto-generated alerts annotation on the list.
Include edited alert annotation on the list.
Add button or checkbox to include alerts with annotation*

**Why is this needed**:

*As I read in the code, It's suggesting to use the alert panel for it but I want something like this which is currently not possible(as what I know and tried to do) and it might be very useful for others also.

![Screenshot (9)](https://user-images.githubusercontent.com/22982363/69139371-5f8c2f80-0ae6-11ea-92c2-46adf6c2d0e3.png)

This is alert markup on graph and values of metrics I have chosen,
So I want to display these values and time somewhere to analyze later. 
Now in the alert panel, we can only see alerting state (ok or alerting and title of alert), not values or specific comment on alert.

![Screenshot (13)](https://user-images.githubusercontent.com/22982363/69140892-7b450500-0ae9-11ea-9ea4-b1c806913cd8.png)
![Screenshot (12)](https://user-images.githubusercontent.com/22982363/69141639-33bf7880-0aeb-11ea-85f8-50550e4cd3d2.png)

As shown above, if you try to edit the alert annotation pop up, it shows blank description and tag field. This will create an annotation but It will lose the values. I want to create an annotation on alert with description and values preserved. And Currently, whether you create an annotation on this alert or not it is not displayed on the annotation list. It shows only annotations that are manually created.
Also for OK state, if you edit or just mouse over, no insights you are getting.

Now there is state history to see some insights of alerts but its not good way to see it by going into the setting of the panel and then click on state history ( I hope you know how to)

In short, there is no good way to see alerts with specific comments, values and time when its generated on the dashboard itself. 

Alerting and Notification must require some improvement. As this module is almost stopped to progress.

I hope my description is easy to understand, if not, please ask me for clarification.
As I can see AnnoList is about to be part of Grafana, these improvements will be great.
If I am missing something or out of my knowledge then please correct me.

**Comment by ZealousMacwan** [comment]
@ryantxu Please look into this issue/feature

**Comment by ZealousMacwan** [comment]
@marefr @torkelo Requesting to prioritize feature-request #20469 https://github.com/grafana/grafana/issues/20469

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). I'm closing feature requests that have seen limited from the community and/or the Grafana team. 

Should this feature still be of interest to you, please re-create it in our [Discussions area](https://github.com/grafana/grafana/discussions). In doing so the community can continue to contribute to exploring, confirming and sharing their use cases (and more).

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Should you wish to connect to discuss anything Grafana-related, I have an open door policy and [my contact details are on my GitHub profile page](https://github.com/pkolyvas).
